modules/light-clients/07-tendermint/types/store.go

```diff
@@ -59,29 +58,14 @@ func SetConsensusState(clientStore sdk.KVStore, cdc codec.BinaryCodec, consensus
 
 // GetConsensusState retrieves the consensus state from the client prefixed
 // store. An error is returned if the consensus state does not exist.
```

Comment:
godoc needs to be updated. Should we make this function private?

Reply:
Good spot

Reply:
Making private makes a bit more awkward for testing. We would actually need to remove the test function for this.
`getConsensusState` will still get coverage from a number of other tests but testing the panic scenarios then isn't really possible.

What do you think? I don't really have any strong feelings about making it public/private

Reply:
lets leave for now